### ISSUE TITLE
Process doesn't start if installed with snap

### ISSUE THREAD

**Issue by drozel** [IsmaelMartinez/teams-for-linux#224]
Snap package starts on my computer only once. If i close Teams (via quit) it doesn't start anymore until i do `snap remove/snap install`. I mean, no windows, no any reaction.

I've checked `ps -aux`, the process does really disappear after quit and doesn't appear after i try to start it again.

It works if installed as .deb, but i have another issue (#171) there

 - OS: Ubuntu 19.04
 - Installation package: snap
 - Version: tried 0.4, 0.6, 0.6.6

**Comment by IsmaelMartinez** [comment]
Look for apparmour in the closed issues and/or use an alternative package
until I can put more info

**Comment by IsmaelMartinez** [comment]
https://github.com/IsmaelMartinez/teams-for-linux/issues/187#issuecomment-516311386
